src/overlays/actors/ovl_Obj_Jg_Gakki/z_obj_jg_gakki.c

```diff
@@ -28,14 +27,41 @@ const ActorInit Obj_Jg_Gakki_InitVars = {
     (ActorFunc)ObjJgGakki_Draw,
 };
 
-#endif
+extern AnimationHeader D_0601B1E8;
+extern SkeletonHeader D_0601B210;
 
-extern UNK_TYPE D_0601B1E8;
+void ObjJgGakki_Init(Actor* thisx, GlobalContext* globalCtx2) {
+    GlobalContext* globalCtx = globalCtx2;
+    ObjJgGakki* this = THIS;
+    f32 frame_count = Animation_GetLastFrame(&D_0601B1E8);
```

Comment:
```suggestion
    f32 frameCount = Animation_GetLastFrame(&D_0601B1E8);
```
We use camel case for our temps instead of snake case. You'll need to change all instances of this temp too